pkg/apis/batch/types.go

```diff
@@ -533,6 +533,23 @@ const (
 	JobFailureTarget JobConditionType = "FailureTarget"
 )
 
+type JobReasonType string
```

Comment:
is anything using these definitions in the internal API package? if not, perhaps avoid the duplication and just define in v1?

Reply:
I realize I was mistaken. I’ll drop these values from this location.